cluster/src/main/scala/com/linkedin/norbert/norbertutils/Clock.scala

```diff
@@ -41,3 +41,7 @@ object SystemClock extends Clock {
 object SystemClockComponent extends ClockComponent {
```

Comment:
(not your code) I think this can be removed since it is not being used. All we need is the `trait`